test/demo-logger-using-pvc/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM centos:7.9.2009
```

Comment:
Can we rename folder to following.
test/demo-logging-to-pvc/Dockerfile

Better, think of a shorter name, instead of demo-logging-to-pvc, but it should still give clear picture too. if not, long name we will go ahead with

Reply:
renamed to `logging-to-pvc-demo`, but still huge will change if I find better.

Reply:
moved and renamed to `/experiments/logging/local`.